tests/scripts/terraform_install.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -euxo pipefail
+
+apt-get install -y unzip
+curl https://releases.hashicorp.com/terraform/${TF_VERSION}/terraform_${TF_VERSION}_linux_amd64.zip > /tmp/terraform.zip
```

Comment:
Should we add all these packages to the docker image. The same with validate vagrantfile, so we don't need to download them on each CI run?